script.js

```diff
@@ -1,4 +1,88 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+// Player One clicks submit
+// Roll two dices and show value
+// Player One chooses dice order
+// Player Two clicks submit
+// Roll two dices and show value
+// Player Two chooses dice order
+// Evaluate higher combined score and output result. Reset game
+
+var gameMode = "playerOneRoll";
```

Comment:
Since "playerOneRoll" is a string you can write it normally instead of camelCasing. So it can be "player one roll"